### ISSUE TITLE
Layout/Tab ignores tabs that are not indentation

### ISSUE THREAD

**Issue by bquorning** [bbatsov/rubocop#5429]
It seems that Layout/Tab will only report on tabs when they are part of a line’s indentation – although [the documentation](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/Tab) says that “this cop checks for tabs inside the source code.”

# This cop checks for tabs inside the source code.

## Expected behavior

In my test file, there is a tab between `puts` and `42`, which I expect Layout/Tab to detect.

```rb
# frozen_string_literal: true

puts	42
```

## Actual behavior

No offense is reported.

## Steps to reproduce the problem

Run `rubocop` on the file above.

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.52.1 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin15)
```
